### PR TITLE
Added my custom PCB - Icarus

### PR DESCRIPTION
**Pull request by Babyyoda777**
Added the variant and board files for my PCB, Icarus as identifier number 84. Verified the firmware works on my boards :)

### CODE REVIEW

**Comment by fifieldt** [comment]
Hi @Babyyoda777 , welcome to meshtastic and congratulations on sending your first patch :)

File: src/mesh/generated/meshtastic/mesh.pb.h [hunk 1]
```diff
@@ -212,6 +212,9 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_MS24SF1 = 82,
     /* Lilygo TLora-C6 with the new ESP32-C6 MCU */
     meshtastic_HardwareModel_TLORA_C6 = 83,
+    /* Icarus custom PCB by Muhammad Shah based on ESP32-S3 and RA-01SH */
+    meshtastic_HardwareModel_ICARUS = 84,
```
Comment (thebentern) [inline src/mesh/generated/meshtastic/mesh.pb.h:216]: This is a generated file. You need to use the DIY hardware model

File: variants/icarus/platformio.ini
Comment (thebentern) [inline variants/icarus/platformio.ini:1]: Please add this, since this will not be added to the official build assets:
board_level = extra

Comment (Babyyoda777) [reply variants/icarus/platformio.ini:1]: done :)

File: src/mesh/generated/meshtastic/mesh.pb.h (hunk 1 above)
Comment (Babyyoda777) [reply src/mesh/generated/meshtastic/mesh.pb.h]: also removed and any using diy

File: protobufs
Comment (thebentern) [inline protobufs:1]: Can you also undo protobuf submodule ref change. Thanks :-)

Comment (Babyyoda777) [reply protobufs:1]: yep I think I did that already

Comment (Babyyoda777) [reply protobufs:1]: right now the only files I added/edited are my variant.h, pins_arduino.h, platformio.ini and board.json

Comment (fifieldt) [reply protobufs:1]: Let us know if you need git help - we understand for many it's the first time dealing with submodules.

Comment (Babyyoda777) [reply protobufs:1]: Yep! I'll probably just close and redo with the updated 2.5.8 firmware :)

**Pull request closed without merging**